runcheckmigration

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+ checkmigrations --migration-path ./django_ipam/migrations/ || exit 1
+ checkmigrations --migration-path ./tests/sample_ipam/migrations/ || exit 1
```

Comment:
it would have been perfect this time if it wasn't for the whitespace in the beginning of these previous lines, please remove that